pages/windows/logoff.md

```diff
@@ -0,0 +1,19 @@
+# logoff
+
+> Terminate a login session.
+
+- Terminate the current session:
+
+`logoff`
+
+- Terminate a session by its name or id:
+
+`logoff {{session_name|session_id}}`
+
+- Terminate a session on a specific server connected through RDP:
+
+`logoff /server:{{servername}}`
+
+- Terminate a session within a server or virtual machine (session id is required):
+
+`logoff /vm {{session_id}}`
```

Comment:
So if my machine is a VM, I need this extra flag?

Reply:
I'm going to remove this, it's very rarely used and probably not necessary in a tldr page. :+1: